src/main/java/org/gridsuite/directory/server/repository/DirectoryElementEntity.java

```diff
@@ -90,4 +96,10 @@ public boolean isAttributesUpdatable(@NonNull ElementAttributes newElementAttrib
             Objects.isNull(newElementAttributes.getLastModificationDate()) &&
             Objects.isNull(newElementAttributes.getLastModifiedBy());
     }
+
+    public DirectoryElementEntity stashElement(boolean stashed, LocalDateTime stashDate) {
+        this.stashDate = stashDate;
+        this.stashed = stashed;
+        return this;
+    }
```

Comment:
why ? both method will update the same variable (stashed, stashDate) and the method return this because it is used in restoreElements and stashElements (same logic is used in method update in the same class)